requirements.txt

```diff
@@ -9,3 +9,4 @@ pandas
 moto
 boto3
 croissant-ml
+marshmallow
```

Comment:
not needed, argschema already pulls it in.

Reply:
Great, had an issue with my ide recognizing it for some reason. Removed now.